fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/common/BulkDataUtils.java

```diff
@@ -204,7 +153,7 @@ private static int getFhirResourceFromBufferReader(BufferedReader resReader, int
                 try {
                     fhirResources.add(FHIRParser.parser(Format.JSON).parse(new StringReader(resLine)));
                     exported++;
-                    if (exported == Constants.IMPORT_NUMOFFHIRRESOURCES_PERREAD) {
+                    if (exported == adapter.getImportNumberOfFhirResourcesPerRead(null)) {
```

Comment:
Maybe change to ">=" just in case somewhere/somehow getImportNumberOfFhirResourcesPerRead
 returned a different value and we loop forever.

Reply:
it is a constant underneath.  However, it could change based on the interface.  Many thanks for the catch.

Reply:
Fixed